api/unauthorized.ts

```diff
@@ -0,0 +1,5 @@
+import { VercelRequest, VercelResponse } from "@vercel/node";
```

Comment:
Not a big deal but any reason why the auth.ts and unauthorized.ts are not within a folder

Reply:
they have diff purposes and no other endpoints I could group either of them with. Just design decision, doesn't really matter